supportedBackends/tes/src/main/scala/cromwell/backend/impl/tes/TesTask.scala

```diff
@@ -32,6 +35,16 @@ final case class TesTask(jobDescriptor: BackendJobDescriptor,
   private val workflowDescriptor = jobDescriptor.workflowDescriptor
   private val workflowName = workflowDescriptor.callable.name
   private val fullyQualifiedTaskName = jobDescriptor.taskCall.fullyQualifiedName
+  private val workflowExecutionIdentityConfig: Option[WorkflowExecutionIdentityConfig] =
+    configurationDescriptor.backendConfig
+      .getAs[String]("workflow-execution-identity")
+      .map(WorkflowExecutionIdentityConfig)
+  private val workflowExecutionIdentifyOption: Option[WorkflowExecutionIdentityOption] =
```

Comment:
such a nitpick, this seems to be an ever-so-subtle typo that might hurt someone not using autocomplete...

`workflowExecutionIdentifyOption` -> `workflowExecutionIdentityOption`